index.html

```diff
@@ -0,0 +1,206 @@
+    <link rel="stylesheet" href="./assets/css/mdb.min.css">
+    <link rel="stylesheet" href="./assets/css/bootstrap.css">
+    <link rel="stylesheet" href="./assets/css/wickedcss.min.css">
+    <link rel="stylesheet" href="./assets/css/main.css">
+    <link rel="stylesheet" href="./assets/css/animations.css">
+    <link rel="stylesheet" href="./assets/css/reset.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/animejs/2.0.2/anime.min.js"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin=""> 
+    <!-- material ui cdn icons -->
+    <link href="https://fonts.googleapis.com/css2?family=Material+Icons" rel="stylesheet">
+    <!-- link to fontawesome library -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+        <!-- link to lato font family -->
+    <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,300&amp;display=swap"
+    rel="stylesheet">
+    <title>Events254 Arena</title>
+</head>
+<body>
+  <header id="header">
+    <!-- topmost small navigation bar start -->
+    <nav class="small-nav bg-dark color-white-text">
+      <div class="fbt">
+        <a href="https://facebook.com">
+          <i class="fab fa-facebook-f" aria-hidden="true"></i>
+        </a>
+        <a href="http://twitter.com">
+          <i class="fab fa-twitter" aria-hidden="true"></i>
+        </a>
+      </div>
+      <ul class="small-bar-actions">
+        <li><button class="color-white-text">Logout</button></li>
+        <li><button class="color-white-text">Mypage</button></li>
+        <li><button class="color-white-text">English</button></li>
+      </ul>
+    </nav>
+<!-- topmost small navigation bar end -->
+<!-- toggle button on mobile screens and smaller devicess -->
+    <button type="button" class="toggle-btn color-dark-text">
+      <img class="bars-fw1 bars-fw1-fs" src="./assets/images/iconmonstr-menu-1.svg" alt="harmburger" aria-hidden="true">
+    </button>
+<!-- main navigation bar -->
+    <nav class="lg-navbar bg-white">
+      <a class="namm-logoo-a" href="./"><img class="namm-logoo" src="./assets/images/logo.png"
+          alt="Events254 Arena"></a>
+
+      <ul class="lg-nav-link">
+        <li><a class="nav-links" href="./about.html">About</a></li>
+        <li><a class="nav-links" href="#program">Program</a></li>
+        <li><a class="nav-links" href="#join">Join</a></li>
+        <li><a class="nav-links" href="#partners">Sponsor</a></li>
+        <li><a class="nav-links" href="#speakers">Guest</a></li>
+        <li><a class="nav-links join-namm-btn" href="ticket_purchae.html">Events254 Arena</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  
+   <!-- start main section here -->
+    <section class="main">
+      
+<!-- start hero section -->
+<section id="namm-2020-news" class="headline">
```

Comment:
- Good job adding animations to your website 👍🏿 

- [ ] Kindly add more spacing between your navigation bar and main section, this will make your website have similar padding to the Behance design

Reply:
I have  fixed that thanks